src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -268,21 +269,15 @@ function RemoteFunctions(experimental) {
                 "box-shadow": "0 0 1px #fff",
                 "box-sizing": "border-box"
             };
+            
+            var mergedStyles = Object.assign({}, stylesToSet,  config.remoteHighlight.stylesToSet);
```

Comment:
Actually as they are remote functions that are run on the browser, we can just use `Object.assign` right?